sw/device/tests/BUILD

```diff
@@ -998,6 +1085,9 @@ opentitan_functest(
 opentitan_functest(
     name = "usbdev_test",
```

Comment:
I think this test should probably be restricted to the Verilator target: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/tests/usbdev_test.c;l=95

Reply:
Thanks I didn't know that